docs/source/user_guide/installation/index.rst

```diff
@@ -50,8 +50,152 @@ Following GNU/Linux and macOS solvers are optionally available:
+as optional extras (e.g.,``pybamm[dev,plot]``). All optional dependencies can be installed with ``pybamm[all]``,
+and specific sets of dependencies are listed in the sections below.
+
+.. _install.plot_dependencies:
+
+Plot dependencies
+^^^^^^^^^^^^^^^^^
+
+Installable with ``pip install "pybamm[plot]"``
+
+=========================================================== ================== ================== ===================================================================================================================================================================================
+Dependency                                                  Minimum Version    pip extra          Notes
+=========================================================== ================== ================== ===================================================================================================================================================================================
+`imageio <https://imageio.readthedocs.io/en/stable/>`__     2.9.0              plot               Handles diverse image data formats, including animated, volumetric, and scientific formats.
+`Matplotlib <https://matplotlib.org/stable/>`__             2.0.0              plot               To plot various battery models, and analyzing battery performance.
+=========================================================== ================== ================== ===================================================================================================================================================================================
+
+.. _install.docs_dependencies:
+
+Docs dependencies
+^^^^^^^^^^^^^^^^^
+
+Installable with ``pip install "pybamm[docs]"``
+
+================================================================================================= ================== ================== ===================================================================================================================================================================================
+Dependency                                                                                        Minimum Version    pip extra          Notes
+================================================================================================= ================== ================== ===================================================================================================================================================================================
+`sphinx <https://www.sphinx-doc.org/en/master/>`__                                                1.5.0              docs               Sphinx makes it easy to create intelligent and beautiful documentation.
+`pydata-sphinx-theme <https://pydata-sphinx-theme.readthedocs.io/en/stable/>`__                   -                  docs               A clean, Bootstrap-based Sphinx theme
```

Comment:
Replace the `-` by `\-` so it renders as a dash

Reply:
Done. 🙂